src/parsers/babel_parser.js

```diff
@@ -171,3 +170,34 @@ export const parse = (file: string, data: ?string): ParseResult => {
 
   return parseResult;
 };
+
+export const plugins = [
+  'asyncGenerators',
+  'bigInt',
+  'classPrivateMethods',
+  'classPrivateProperties',
+  'classProperties',
+  'doExpressions',
+  'dynamicImport',
+  'estree',
+  'exportDefaultFrom',
+  'exportNamespaceFrom', // deprecated
+  'functionBind',
+  'functionSent',
+  'importMeta',
+  'jsx',
+  'logicalAssignment',
+  'nullishCoalescingOperator',
+  'numericSeparator',
+  'objectRestSpread',
+  'optionalCatchBinding',
+  'optionalChaining',
+  'partialApplication',
+  'throwExpressions',
+  'topLevelAwait',
+];
```

Comment:
looks like there is something missing from the [babel plugin list](https://github.com/babel/babel/blob/928b9f8c9518284eac6d0598633f2ec373fc6d0c/packages/babel-parser/typings/babel-parser.d.ts#L97), is it intentional?

Reply:
there are conflicts between various plugins, tried to include all and removed the ones which caused conflicts